### PR TITLE
Add Redis backup restore doc for KubeStash

### PR DESCRIPTION
**Pull request by Neaj-Morshad-101**

### CODE REVIEW

File: docs/guides/redis/backup/kubestash/application-level/index.md [hunk 1]
```diff
@@ -0,0 +1,658 @@
...
+
+
+#### Verify Restored Redis Manifest:
+
+In this section, we will verify whether the desired `Redis` database manifest has been successfully applied to the cluster.
+
+```bash
+$ kubectl get redis -n dev 
+NAME              VERSION   STATUS   AGE
+sample-redis   16.1      Ready    9m46s
```
Comment (AbdullahAlShaad) [inline docs/guides/redis/backup/kubestash/application-level/index.md]: Wrong Version

Comment (AbdullahAlShaad) [reply docs/guides/redis/backup/kubestash/application-level/index.md]: Create Redis in dev namespace?

**Review by AbdullahAlShaad — CHANGES_REQUESTED** [review]

File: docs/guides/redis/backup/kubestash/application-level/index.md (hunk 1 above)
Comment (Neaj-Morshad-101) [reply docs/guides/redis/backup/kubestash/application-level/index.md]: Yes. 
```
  manifestOptions:
    restoreNamespace: dev
```

File: docs/guides/redis/initialization/using-script.md [hunk 2]
```diff
@@ -428,7 +428,7 @@ kubectl delete ns demo
 
 ## Next Steps
 
-- [Backup and Restore](/docs/guides/redis/backup/overview/index.md) Redis databases using Stash.
+- [Backup and Restore](/docs/guides/redis/backup/stash/overview/index.md) Redis databases using Stash.
```
Comment (AbdullahAlShaad) [inline docs/guides/redis/initialization/using-script.md]: Give KubeStash link instead of Stash

**Review by AbdullahAlShaad — CHANGES_REQUESTED** [review]

**Review by AbdullahAlShaad — APPROVED** [review]
LGTM!

**Pull request merged**